domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/arbeidsforhold/impl/InntektsmeldingFilterYtelse.java

```diff
@@ -16,11 +16,11 @@ public interface InntektsmeldingFilterYtelse {
      * Returnerer påkrevde inntektsmeldinger etter ytelsesspesifikke vurdering og
      * filtrering
      */
-    <V> Map<Arbeidsgiver, Set<V>> filtrerInntektsmeldingerForYtelse(BehandlingReferanse referanse, Map<Arbeidsgiver, Set<V>> påkrevde);
+    <V> Map<Arbeidsgiver, Set<V>> krevKunIMFraArbeidsforholdDetErSøktSvpFor(BehandlingReferanse referanse, Map<Arbeidsgiver, Set<V>> påkrevde);
```

Comment:
Svp i metodenavn er ikke helt ønskelig - enten ha ytelsesspesifikk impl som nå eller dra if ytelse inn i ImRegTjeneste

Foreslår "søknadsFilter" - da vil FP si alle og SVP de som er omsøkt og "aktiveArbeidsforholdFilter". Da sier det hva man filtrerer for. Evt "påkrevdFraSøknadFilter"

Reply:
Jeg mener at det hadde vært bedre med if(ytelse) her for lesbarhet siden det kun er foreldrepenger og svangerskapspenger som krever inntektsmelding. Men endrer slik du sier så får det eventuelt bli senere. Det er fortsatt rom for forbedringer her